src/reporter.ts

```diff
@@ -305,7 +305,7 @@ export default class SauceReporter implements Reporter {
           break;
         }
 
-        const filename = `${path.basename(path.dirname(attachment.path || ''))}-${path.basename(attachment.path || '')}`;
+        const filename = path.basename(attachment.path || '');
```

Comment:
I think there's some bug in the reporter, at least logically, a bit further down:

```
        } else if (attachment.body) {
          assets.push({
            filename,
            data: stream.Readable.from(attachment.body),
          });
        }
```

If `body` is specified, `path` is likely undefined, so `filename` is an empty string in this case.
In that case, we should use `attachment.name`, right?